src/main/java/com/novi/backend_spring_boot_tech_it_easy_controller/controllers/TelevisionsController.java

```diff
@@ -0,0 +1,38 @@
+package com.novi.backend_spring_boot_tech_it_easy_controller.controllers;
+
+import com.novi.backend_spring_boot_tech_it_easy_controller.models.TelevisionRequest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+
+@RequestMapping("/api/televisions")
+public class TelevisionsController {
+
+    @GetMapping
+    @ResponseBody
+    public String getAllTelevisions() {
+        return "List Televisions";
+    }
+
+    @GetMapping("/{model}")
```

Comment:
Je hebt hier voor de naam van je pathvariable gekozen voor "model". Echter heeft je Television geen "model" veld. Gebruikelijk is ook om de identifier van je entity te kiezen, en dat heet bij jou Telvision toepasselijk "id"